ajax/libs/vue-functional-calendar/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "version": "2.8.69",
```

Comment:
```suggestion
```
Please don't include the `version` attr, the auto-update bot will populate this when it pulls in versioned files.